notary-btc-integration-test/src/main/kotlin/integration/helper/BtcIntegrationHelperUtil.kt

```diff
@@ -239,6 +239,7 @@ class BtcIntegrationHelperUtil(peers: Int = 1) : IrohaIntegrationHelperUtil(peer
     fun sendBtc(address: String, amount: Int, confirmations: Int = 6) {
         logger.info { "Send $amount BTC to $address" }
         rpcClient.sendToAddress(address = address, amount = BigDecimal(amount))
+
```

Comment:
:suspect:

Reply:
This line has been brutally demolished from the file.